packages/tools/src/comparison/compare-runs.ts

```diff
@@ -0,0 +1,257 @@
+import { output, RequestInput } from '@covid-modeling/api'
+import { assert } from 'chai'
+import * as fs from 'fs'
+import * as path from 'path'
+import * as pino from 'pino'
+import * as unzipper from 'unzipper'
+import { logger } from '../logger'
+import {
+  enforceOutputSchema,
+  enforceRunnerInputSchema,
+} from '@covid-modeling/model-runner/dist/src/schema'
+import { parse } from '@covid-modeling/mrc-ide-covidsim/dist/src/imperial-params'
```

Comment:
Since this file is imperial-specific, we should rename it to `compare-imperial-runs.ts`.

Reply:
It is capable of comparing any runs, it just has additional functionality for Imperial.